doc/experiment.rst

```diff
@@ -1,327 +1,423 @@
+and capable of performing inference and script evaluation on feature
+store data. Any SmartSim ``Model`` or ``Ensemble`` model can connect to the
+``Orchestrator`` via the :ref:`SmartRedis<SmartRedis Client Library Hook>`
+client library to transmit data, execute ML models, and execute scripts.
+
+**SmartSim offers two types Orchestrator deployments:**
+
+* :ref:`Clustered Orchestrator <Clustered Orchestrator>`
+* :ref:`Colocated Orchestrator <Colocated Orchestrator>`
+
+Clustered Orchestrator
+^^^^^^^^^^^^^^^^^^^^^^
+The orchestrator can be composed of one or more in-memory database shards that are spread
+across one or more compute nodes.
+The multiple compute hosts memory can be used together to store data.
+Users do not need to know how the data is stored in a clustered
+configuration and can address the cluster with a SmartRedis client
+like a single block of memory using simple put/get semantics in SmartRedis.
+The database shards communicate with each other via TCP/IP in the driver script and application.
+SmartRedis will ensure that data is evenly distributed among all nodes in the cluster.
+
+Clustered Deployment Diagram
+""""""""""""""""""""""""""""
+During clustered deployment, a SmartSim ``Model`` (the application) runs on separate
+compute node(s) from the database node(s).
+A clustered database is optimal for high data throughput scenarios
+such as online analysis, training and processing.
+
+Below is an image illustrating communication
+between a clustered orchestrator and a
+multi-node model. In the Diagram, an instance of the application is
+running on each application compute node. A single SmartRedis client is initialized with
+the clustered database address and used to communicate with the application's compute nodes.
+Data is streamed from the application compute nodes to the sharded database via the client.
+
+.. figure::  images/clustered-orc-diagram.png
+
+Initialize a Clustered Orchestrator
+"""""""""""""""""""""""""""""""""""
+To create an ``Orchestrator`` that does not share compute resources with other
+SmartSim entities, use the ``Experiment.create_database()`` factory method.
+Specifying the parameter `db_nodes` as greater than or equal to 1 will determine
+whether your database is multi-sharded or single-sharded.
+This factory method returns an initialized ``Orchestrator`` object that
+gives you access to functions associated with the :ref:`Orchestrator API<orc_api>`.
+
+Colocated Orchestrator
+^^^^^^^^^^^^^^^^^^^^^^
+An ``Orchestrator`` can be created to share the compute node(s)
+and resources with a SmartSim ``Model``. In this case, the database
+is deployed on the same compute hosts as a Model instance
+defined by the user. In this deployment, the database is not connected
+together in a cluster and each shard of the database is addressed
+individually by the processes running on that compute host.
+If the SmartSim ``Model`` spans more than one
+compute node, the colocated database will also span all of the
```

Comment:
It's a little confusing to say that the co-located database "spans" the compute nodes as it suggests that the each database is somehow connected to the others which is not correct. More accurate to say, "essentially, this means that you have N independent databases, where N is the number of compute nodes assigned to the applicaton"